guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -101,6 +101,24 @@ You must migrate a self-registered {Project} to the Red Hat Content Delivery Net
 To migrate a self-registered {Project} to the CDN, see {UpgradingDocURL}Upgrading_Server_upgrade-guide[Upgrading {ProjectName}] in the _Upgrading and Updating {ProjectName}_ guide.
 endif::[]
 
+ifdef::satellite[]
+.Considerations for Upgrades of {Project} to Future Versions
+
+Before you begin the upgrade to {Project} 6.10, which includes an upgrade to Pulp 3 on {ProjectServer}s, it is highly recommended that you complete pre-migration of Pulp content.
+If you perform no migrations on 6.9, they will all be performed during the upgrade to 6.10.
+For smaller content setups, or if you can afford longer downtimes, you can proceed without the pre-migration.
+
+For {SmartProxies}, you can choose to re-deploy new 6.10 {SmartProxies} rather than to upgrade.
+
+For future upgrades following {Project} 7.0, you will be required to upgrade the operating system from RHEL 7 to RHEL 8 on your {ProjectServer}s and {SmartProxies}.
+You can upgrade the operating system in-place or through a cloning process. The latter includes migration of all data, configuration, and synced content.
```

Comment:
@Lennonka Ideally, you only have one sentence per line. See [conventions](https://github.com/theforeman/foreman-documentation/tree/master/guides#conventions).